src/curvefit/solvers/solvers.py

```diff
@@ -33,6 +74,9 @@ def get_model_instance(self):
         else:
             raise ModelNotDefinedError()
 
+    def set_options(self, options: SolverOptions):
+        self.options = options
+
```

Comment:
maybe we also need something like this for composite solver.
```
def set_options(self, options):
    if is_solver_defined():
           self.solver.set_options(options)

Reply:
Sure! Done in 
https://github.com/ihmeuw-msca/CurveFit/pull/86/commits/c26c46cda71a7e1592bcddb63241d2a85f873ec8